java/client/src/com/thoughtworks/selenium/condition/DefaultConditionRunner.java

```diff
@@ -173,7 +173,7 @@ public void conditionFailed(ConditionRunner.Context context, Condition condition
     }
 
     protected void log(String message) {
-      logger.info(new Date() + " - " + message);
+      logger.finest(new Date() + " - " + message);
```

Comment:
This method is designed to allow the condition runner to tell users something interesting. The current `INFO` log level is correct.